mail/messages.go

```diff
@@ -1,81 +1,81 @@
 package mail
 
 import (
+	"bytes"
+	"html/template"
 	"memberserver/config"
 
 	log "github.com/sirupsen/logrus"
 )
 
-func SendGracePeriodMessageToLeadership(address string) {
+func SendGracePeriodMessageToLeadership(recipient string, member interface{}) {
 	infoAddress := "info@hackrva.org"
+	SendTemplatedEmail("pending_revokation_leadership.html.tmpl", infoAddress, "hackRVA Grace Period", member)
+}
 
-	conf, _ := config.Load()
-	if !conf.EnableInfoEmails {
-		return
-	}
+func SendGracePeriodMessage(recipient string, member interface{}) {
+	SendTemplatedEmail("pending_revokation_member.html.tmpl", recipient, "hackRVA Grace Period", member)
+}
 
-	mp, err := Setup()
-	if err != nil {
-		log.Errorf("error setting up mailprovider when attempting to send email notification")
-	}
+func SendRevokedEmail(recipient string, member interface{}) {
+	SendTemplatedEmail("access_revoked.html.tmpl", recipient, "hackRVA Grace Period", member)
+}
 
-	if len(conf.EmailOverrideAddress) > 0 {
-		infoAddress = conf.EmailOverrideAddress
-	}
+func SendRevokedEmailToLeadership(recipient string, member interface{}) {
+	SendTemplatedEmail("access_revoked_leadership.html.tmpl", recipient, "hackRVA Grace Period", member)
+}
 
-	mp.SendSMTP(infoAddress, address+": hackrva grace period", address+" membership is in a grace period.  \n\nIf a payment isn't received, their membership will be revoked.")
+func SendIPHasChanged(newIpAddress string) {
+	recipient := "info@hackrva.org"
+	model := struct {
+		IpAddress string
```

Comment:
Nit picky but would it be alright if we could rename this to be `ipModel` instead so that it's clearer?

Reply:
Renamed